### DISCUSSION TITLE
How to run Capacitor webView with https in IOS

### DISCUSSION THREAD

**Discussion by sibaprasadsahoo** [ionic-team/capacitor#3714] [Q&A] [answered]
I already tried below: 
In the capacitor.config.json in the server section:
“server”: {
“hostname”: “localhost:3000”,
“iosScheme”: “https”,
“androidScheme”: “https”
},

In android, The Capacitor webview runs with https:// protocol.

In iOS , The capacitor webview runs with capacitor://,   but I want it to runs with https

Reason, we are using amazon sign in url, which needs https redirect url after successful signin.

**Comment by jcesarmobile** [comment, marked as answer]
It's not possible to serve local assets using https scheme, https scheme is reserved by the WKWebView for remote urls.

**Reply by Sampath-Lokuge** [reply]
@sibaprasadsahoo What kind of error do you have now on an iOS device? I also have this issue here: https://github.com/ionic-team/capacitor/discussions/3712#discussioncomment-107874 For It is with Firebase Email, Pw login on an iOS device with Native build.
